content/login-from-live-disk.md

```diff
@@ -34,7 +36,7 @@ lsblk
 
 This will show you the name of the main internal drive, which will have 4 partitions (Pop!\_OS), or 3 (Ubuntu) on it.  We will be working with the 3rd partition.  If the main drive is an NVMe drive, it will be called `/dev/nvme0n1p3` (p2 on Ubuntu) and if the drive is a SATA or regular M.2 drive, it will be called `/dev/sda3` (sda2 on Ubuntu).
 
-**NOTE:** The rest of these instructions assume a Pop!\_OS install, for partition labelling. The instructions are otherwise the same. Change the partition number accordingly.
+>**Note:** The rest of these instructions assume a Pop!\_OS install, for partition labelling. The instructions are otherwise the same. Change the partition number accordingly.
```

Comment:
I would reword this as:

The rest of these instructions assume partition labeling consistent with a Pop!\_OS install. For an Ubuntu install, change the partition number accordingly; the instructions are otherwise the same.